includes/class-sensei-utils.php

```diff
@@ -2714,6 +2714,34 @@ public static function is_teacher_archive_page() {
 		return false;
 	}
 
+	/**
+	 * Gets the contents of the icon file at assets/images/<name>.svg
+	 * for the given name. Or empty string if file not found.
+	 *
+	 * @param string $name The name of the icon file at "assets/images/<name>.svg".
+	 * @return string The icon markup.
+	 */
+	public static function icon( string $name = '' ) {
```

Comment:
Nice! But it could be a better fit in `Sensei_Assets` or a new `Sensei_Icons` class, this one is already pretty huge :)

Reply:
Moved the method to `Sensei_Assets`. 👍